src/c/admin-list.js

```diff
@@ -29,7 +29,7 @@ window.c.AdminList = (function(m, h, c){
               ]),
               m('#admin-contributions-list.w-container',[
                 list.collection().map(function(item) {
-                  return m.component(c.AdminItem, {builder: args.itemBuilder, actions: args.itemActions, item: item, key: item.key});
+                  return m.component(c.AdminItem, {builder: args.itemBuilder, actions: args.itemActions, item: m.prop(item), key: item.key});
```

Comment:
By what I've seen, we don't even need the item objects to be propified. The only thing we need is the updated resource coming back updated and completely described from the patch.